src/main/java/com/treasuredata/client/model/TDExportResultJobRequest.java

```diff
@@ -0,0 +1,47 @@
+package com.treasuredata.client.model;
+
+import org.immutables.builder.Builder;
+import org.immutables.value.Value;
+
+@Value.Style(typeBuilder = "TDExportResultJobRequestBuilder")
+public class TDExportResultJobRequest
+{
+    private final String jobId;
+    private final String result;
+
+    private TDExportResultJobRequest(String jobId, String result)
```

Comment:
To keep consistent with `TDJobRequest`, can we rename 2nd parameter name with `resultOutput`? `TDJobRequest` has `resultOutput` field. The field has same meaning with the 2nd parameter. What do you think?

This is just comment. I also thought that we'd better to make the 2nd parameter `Optional` like `TDJobResult.resultOutput`  but, It's ok for me to use just `String` now.

Reply:
@muga 

Thank you for your comment.

> To keep consistent with TDJobRequest, can we rename 2nd parameter name with resultOutput?

I think It's reasonable. I will change it like that.

>  we'd better to make the 2nd parameter Optional like TDJobResult.resultOutput

It seems that in model classes implementation, `String` class is used for required text parameters, and `Optional` class is used for not-required(optional) parameters.  
And `resultOutput` in `TDExportResultJobRequest` is a required parameter in API specification, though the one in `TDJobRequest` is optional parameter.
So I think it's better to use String here to avoid misleading, even if consistency in type is missed.